improver_tests/standardise/test_RegridLandSea.py

```diff
@@ -49,138 +46,10 @@
 WARNING_TYPES = [FutureWarning]
 
 
-class Test__init__(unittest.TestCase):
-    """Test initialisation"""
-
-    def test_default(self):
-        """Test initialisation with default options"""
-        plugin = StandardiseGridAndMetadata()
-        self.assertEqual(plugin.regrid_mode, "bilinear")
-        self.assertEqual(plugin.extrapolation_mode, "nanmask")
-        self.assertIsNone(plugin.landmask_source_grid)
-        self.assertIsNone(plugin.landmask_vicinity)
-        self.assertEqual(plugin.landmask_name, "land_binary_mask")
-
-    def test_error_missing_landmask(self):
-        """Test an error is thrown if no mask is provided for masked
-        regridding"""
-        msg = "requires an input landmask cube"
-        with self.assertRaisesRegex(ValueError, msg):
```

Comment:
The testing of this exception from the `__init__` should potentially still be tested as a possible pathway through the code, even if this test is within the `Test_process` class.